commcare_export/commcare_hq_client.py

```diff
@@ -157,7 +157,14 @@ def _get(resource, params=None):
                 except Exception as e:
                     # for non-verbose output, skip the stacktrace
                     if not logger.isEnabledFor(logging.DEBUG):
-                        logger.error(str(e))
+                        if isinstance(e, requests.exceptions.HTTPError) and response.status_code == 401:
+                            logger.error(
+                                f"#{e}. Please ensure that your CommCare HQ credentials are correct & valid for "
+                                f"the auth-mode passed. Also, Verify that your account has the necessary "
+                                f"permissions to access the DET tool."
```

Comment:
("the DET tool" means "the data export tool tool".) But I'm wondering whether the user knows that Dimagi internally refers to this as "the DET". In help text this command refers to itself as "commcare-export" or "the commcare-export tool". I think this text might be more familiar to a user:
```suggestion
                                f"permissions to use commcare-export."
```

Reply:
Makes sense. Thank you

https://github.com/dimagi/commcare-export/pull/230/commits/5d99505143702c1bc320adec8635a134e01eeebb